inputs/orchestrator_sources_inner:6.json

```diff
@@ -0,0 +1,34 @@
+{
+  "prebuild_plugins": [
+    {
+      "name": "reactor_config"
+    }
+  ],
+  "buildstep_plugins": [
+    {
+      "name": "source_container_build"
+    }
+  ],
+  "postbuild_plugins": [],
+  "prepublish_plugins": [],
+  "exit_plugins": [
+    {
+      "name": "koji_import"
+    },
+    {
+      "name": "koji_tag_build"
+    },
+    {
+      "name": "store_metadata_in_osv3"
+    },
+    {
+      "name": "sendmail"
```

Comment:
I wonder if this is needed at all for these. There will never be imageChange triggers for them, so they'll always be directly-requested.